src/vxworks/mod.rs

```diff
@@ -431,6 +415,16 @@ s_no_extra_traits! {
         pub __ss_pad2  : [::c_char; _SS_PAD2SIZE],
     }
 
+    pub union sa_u_t {
+        pub sa_handler : extern "C" fn(::c_int) -> !,
```

Comment:
These function types need to be `unsafe fn`, and since they can probably be null, need to be behind an `Option<...>`, so 
```suggestion
        pub sa_handler : Option<unsafe extern "C" fn(::c_int) -> !>,
```